x-pack/test/functional/apps/machine_learning/data_visualizer/index_data_visualizer.ts

```diff
@@ -0,0 +1,237 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+interface FieldCard {
```

Comment:
Is this something we could import from the `ML` plugin?

Reply:
Done in https://github.com/elastic/kibana/pull/51832/commits/dfde35a493e596fe1db4d22b8d6dc00e0d14de52